pkg/api/pluginproxy/ds_proxy_test.go

```diff
@@ -126,8 +127,10 @@ func TestDataSourceProxy_routeRule(t *testing.T) {
 		t.Run("When matching route path", func(t *testing.T) {
 			ctx, req := setUp()
 			dsService := datasources.ProvideService(bus.New(), nil, secretsService)
+			tracer, err := tracing.InitializeTracerForTest()
```

Comment:
I wonder if this can be done once at the top-level test function and then tracer instance would be reused in the following sub-tests?